tests/test_suite.html

```diff
@@ -548,7 +548,8 @@
             },
             recipes: {
                 '0': {
-                    name: 'control'
+                    name: 'control',
+                    sampleRate: 0
```

Comment:
@allmywant @dapperdrop - Should we introduce a test for the recipe sample rates failure?

Reply:
Yes I think we should.

Reply:
@kingo55 
Yes we can. But we must wrap the code with try/catch or gulp test will stop running the subsequent tests.

Reply:
OK - good point @allmywant. Please add a test inside try..catch.

Reply:
@kingo55 
Added a test case for this.